### DISCUSSION TITLE
Can we "monitor or hook" the network request flow generated by `webview`?

### DISCUSSION THREAD

**Discussion by JoiLa** [wailsapp/wails#3066] [General]
in Go code,  Can we "monitor or hook" the network request flow generated by 'webview'?  

like middleware that, inject network and custom middleware response content.

**Comment by JoiLa** [comment]
becase so far, i dot't know latest `wails` version has support this feature?


this my only one idea, can  solve webview `CORS` problem  :)

**Reply by ykdiy** [reply]
The front-end calls the go method, and the go initiates the HTTP call

**Reply by ykdiy** [reply]
https://wails.io/zh-Hans/docs/reference/runtime/events

**Reply by JoiLa** [reply]
ah, before already try for this method, but it's not i what want.

so. I find someting `webview` document event,  you can take  a look [link](https://learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/webresourcerequested?tabs=dotnet).

Also, ☕ next version support this feature ?
